src/main/java/com/p6spy/engine/wrapper/StatementWrapper.java

```diff
@@ -246,7 +246,7 @@ public void addBatch(String sql) throws SQLException {
     if (statementInformation.getStatementQuery() == null) {
       statementInformation.setStatementQuery(sql);
     } else {
-      statementInformation.setStatementQuery(sql + statementInformation.getStatementQuery());
+      statementInformation.setStatementQuery(sql + System.lineSeparator() + statementInformation.getStatementQuery());
```

Comment:
`System.lineSeparator()` was only introduced in Java 7, use `System.getProperty("line.separator")` instead